src/popup.js

```diff
@@ -139,22 +228,82 @@ function getUrlParameter(sParam) {
     }
 }
 
-function restoreOriginalContent() {
-    sendMessageToContentScript("restoreOriginalContent", updateStatusCallback);
-}
-
 function toggleSomething() {
     sendMessageToContentScript("toggle");
 }
 
 function reloadPage() {
-    sendMessageToContentScript("reloadPage");
-    window.close();
+    callOnTargetTab(function(tab) {
+        chrome.tabs.reload(tab.id);
+        window.close();
+    });
 }
 
-document
-    .getElementById(ids.restore)
-    .addEventListener("click", restoreOriginalContent);
+function setReloadMessage(message) {
+    document.getElementById(ids.reloadMessage).innerHTML = message;
+}
+
+function toggleHostWithoutScript() {
```

Comment:
I found mention of `Callback` and `Handler` suffixes, and I chose to use `Handler` since it responds to a click from the user. This change brings the risk of sounding as though the method handles scripts.

Its counterpart `toggleHostWithScript` needs documentation too, but it's called both as a handler and on its own. I haven't worked with callbacks much before this, so I'm not familiar with conventions. What do you think makes sense here?